docs/dapp/opl/hyperlane/README.md

```diff
@@ -0,0 +1,63 @@
+---
+description: Hyperlane Bridge
+---
+
+# Hyperlane
+
+[Hyperlane] is a permissionless interoperability protocol that enables seamless
+cross-chain communication for developers. Its unique design allows deployment
+across various blockchain environments, including layer 1 chains, rollups, and
+app-chains, without the need for approvals or intermediaries. This empowers
+developers to build cross-chain applications with full control over their
+operations in a multi-chain ecosystem. 
+
+Features of Hyperlane:
+
+- Modular Interchain Security Modules (ISMs)
+- Warp Routes for token transfers
+- Interchain Accounts (ICA) for cross-chain smart contract interactions
+- Interchain Queries (IQS) for remote data access
```

Comment:
I would omit this since the list can get outdated quickly and it's also not something we systematically test.

```suggestion
```